examples/demo/src/index.tsx

```diff
@@ -1,8 +1,11 @@
-import 'proxy-polyfill';
-
-import * as React from 'react';
```

Comment:
I'm surprised that you don't need this anymore

Reply:
Bump. The default root for a React project in Vite imports `React` for JSX to work. Why is it not necessary here?

Reply:
Well, just run the demo, you'll see it works both in development and production